serverless/installing_serverless/upgrading-serverless.adoc

```diff
@@ -8,6 +8,39 @@ toc::[]
 
 If you have previously installed a Technology Preview version of {ServerlessProductName}, follow the instructions in this guide to upgrade to the latest version.
 
+[id="update-api-group_{context}"]
+== Updating the API group
+
+In {ServerlessProductName} 1.4.0 and newer versions, the `KnativeServing` object's API group changed from `serving.knative.dev` to `operator.knative.dev`.
+You must adjust any of your scripts or applications that rely on the old API group to use the new API group.
+
+The {ServerlessOperatorName} creates a `KnativeServing` custom resource (CR) in the new API group for you.
```

Comment:
I'd say it would be better to mention "custom resource definition" (CRD) instead of CR. 
The Serverless operator creates this new CRD and later the user has to manually create the KnativeServing CR in order to install Knative. The Serverless operator doesn't automatically create KnativeServing CR
Something like this:
"The {ServerlessOperatorName} creates the `knativeserving` custom resource definition (CRD) in the new API group for you."

Reply:
Ah OK. Odd, the release notes have the same content and were already QE'd. I will update here and in the release notes. Thanks!

Reply:
Yeah. That was probably an oversight.